dev-libs/hx711/hx711-2.13.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Small c++ Interface library for the external HX711 chip thru GPIO"
+
+HOMEPAGE="https://github.com/endail/hx711"
+
+SRC_URI="https://github.com/endail/hx711/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+
+SLOT="0"
+
+# for SBC's (such as Raspberry Pi)
+KEYWORDS="~arm ~arm64"
```

Comment:
I hope you have personally tested both these arches. I don't have an arm{,64} device so I can't run them